README.md

```diff
@@ -10,7 +12,20 @@
 [![Slack](https://img.shields.io/badge/%23eventing-white.svg?logo=slack&color=522a5e)](https://knative.slack.com/archives/C9JP909F0)
 [![CII Best Practices](https://bestpractices.coreinfrastructure.org/projects/5913/badge)](https://bestpractices.coreinfrastructure.org/projects/5913)
 
-This repository contains a work-in-progress eventing system that is designed to
+## What is Knative Eventing?
+
+Knative Eventing is a collection of APIs that enable you to use an [event-driven architecture](https://en.wikipedia.org/wiki/Event-driven_architecture) with your applications. You can use these APIs to create components that route events from event producers to event consumers, known as sinks, that receive events. Sinks can also be configured to respond to HTTP requests by sending a response event.
+
+Knative Eventing does not _require_ the use of Knative Serving, but it can be used for serverless workload. However, Knative Eventing works _perfect_ with standard Kubernetes Services as well.
```

Comment:
Maybe rephrase to "Knative Eventing is a standalone component that supports EDA for any type of workload eg. K8s raw deployments, Knative Services etc. No restriction is imposed on what you can connect together via events."

Reply:
+1

Reply:
good points in there. I have updated it